util/libcrypto.num

```diff
@@ -4742,10 +4742,10 @@ EVP_SIGNATURE_free                      4858	3_0_0	EXIST::FUNCTION:
 EVP_SIGNATURE_up_ref                    4859	3_0_0	EXIST::FUNCTION:
 EVP_SIGNATURE_provider                  4860	3_0_0	EXIST::FUNCTION:
 EVP_SIGNATURE_fetch                     4861	3_0_0	EXIST::FUNCTION:
-EVP_PKEY_sign_init_ex                   4862	3_0_0	EXIST::FUNCTION:
+EVP_PKEY_sign_init_ex                   4862	3_0_0	NOEXIST::FUNCTION:
```

Comment:
Is there a reason to mark it as NOEXIST rather than just delete it?

Reply:
It's what the updating scripts do.  It's what they have always done.  I don't *really* know why, possibly for reasons of uncertainty, in case the symbol would come back with the same meaning.

I'm putting that down to "tradition" :wink:

Reply:
It's not like there ever was a release with them

Reply:
That's quite true.  But that's for another PR.  I have this action item on me to change the script so it only assigns numbers at release time (I'm going make the number marker a simple question mark).  At that point, it becomes much easier to automatically remove symbols that disappear and haven't yet been released.  So if you don't mind, save this discussion for then, yeah?

Reply:
sure